flocker/node/script.py

```diff
@@ -333,7 +334,13 @@ def get_configuration(options):
     agent_config = options[u'agent-config']
     configuration = yaml.safe_load(agent_config.getContent())
 
-    validate_configuration(configuration=configuration)
+    # FLOC-1925 remove this.
+    # It doesn't really work at the moment:
+    # https://github.com/ClusterHQ/flocker/pull/1440#issuecomment-105502142
+    # And if we want to support pluggable backends the config validation
+    # has to be pluggable too.
+    # JSON Schema style validation can be done
+    # validate_configuration(configuration=configuration)
```

Comment:
Instead of deleting this, I'd suggest just making it more lenient.  Let it enforce the parts that aren't about the storage driver.  Make the storage driver requirements wide open (a `dataset` section with anything at all in it).